### ISSUE TITLE
[BUG]: Offline issues (error popups) using the suggrsted V3 scheduler on "experimental" Backend

### ISSUE THREAD

**Issue by andreas1974** [ankidroid/Anki-Android#15222]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

I updated the Windows Anki program. There I was advised to change to V3 scheduler. I had to do the same change in Ankidroid to avoid synching issues. So I turned on the experimental backend to be able to select V3 on Ankidroid. After that change, every now and then, I get errors due to the fact that I often use my phone **offline**. Ankidroid never had any issues working offline using the default backend.

### Expected behaviour

I would expect Ankidroid to work without interferring error messages even when I pull the cord to the ethernet adapter (offline).

### Actual behaviour

Connection related error popups. the first one is most common.
![image](https://github.com/ankidroid/Anki-Android/assets/998258/baa8aa68-cbe0-4f86-8066-fbbd117b4770)
_Error
Please check your internet connection.
Error details: error sending request for url (): error trying to connect: dns error: failed to lookup address information: No address associated with hostname.  OK_

![image](https://github.com/ankidroid/Anki-Android/assets/998258/d7219113-0777-461e-9753-2e2b4d463b0b)

### Debug info

```text
AnkiDroid Version = 2.16.5 (953b9bd879269910ee962b520da9705336d8bc2d)

Android Version = 9

ProductFlavor = play

Manufacturer = LGE

Model = LG-H930

Hardware = joan

Webview User Agent = Mozilla/5.0 (Linux; Android 9; LG-H930 Build/PKQ1.190414.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/120.0.6099.144 Mobile Safari/537.36

ACRA UUID = 5b935c08-681b-4ebb-87c5-7701905caf7a

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

Since Oct 2023 support for the V2 scheduler was dropped in the computer version of Ankiweb https://github.com/ankitects/anki/releases/tag/23.10, and I believe I needed to change to V3 in the Ankidroid version as well. Is there any other way to go on? Would you recommend me to downgrade instead?

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by BrayanDSO** [comment]
Fixed in 2.17. But thanks for the report

**Comment by andreas1974** [comment]
> Fixed in 2.17. But thanks for the report

Great, and thank you for you work!

This is a bit off topic then... but would you recommend I try to install 2.17 (alpha) instead of using 2.16.5 with the "experimental" backend waiting for a new version to be released on Google Play?

**Comment by BrayanDSO** [comment]
You can try one of the `parallel` versions at [releases page](https://github.com/ankidroid/Anki-Android/releases) to test the new stuff, which includes making the new backend the default, and uninstall it if you didn't like it. 

Then you can decide if you should be using the alpha or the stable version. Of course, being an alpha, some bugs will appear, but we /try/ to make it as stable as we can. The pending issues for releasing 2.17 to the public are [here](https://github.com/ankidroid/Anki-Android/issues?q=is%3Aopen+is%3Aissue+milestone%3A%222.17+release%22) you want to look at it
